onos-uenib/templates/storageprofile.yaml

```diff
@@ -1,8 +1,8 @@
-# SPDX-FileCopyrightText: 2022 2020-present Open Networking Foundation <info@opennetworking.org>
```

Comment:
I think you're supposed to leave the old copyright there, and add the Intel one on a new line

Reply:
Yes thanks for pointing that out. I copied that from another chart :-P